changelogs/fragments/4797-terraform-complex-variables.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - terrform - adds capability to handle complex variable structures for ``variables`` parameter in the module (https://github.com/ansible-collections/community.general/pull/4797). 
```

Comment:
please add a newline to the end of the file

Reply:
Done